.circleci/config.yml

```diff
@@ -25,9 +25,51 @@ jobs:
             [ "$(cat test-reports/python-keepkey/status)$(cat test-reports/firmware-unit/status)" == "00" ] || exit 1
       - store_test_results:
           path: test-reports
+
+  emulator-publish:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run:
+          name: Update Submodules
+          command: |
+            git submodule update --init --recursive
+      - setup_remote_docker
+      - run:
```

Comment:
You should be able to drop lines 39-53 and replace this with one `docker build` command in the `Publish Emulator` step.